bin/init_exercise.py

```diff
@@ -0,0 +1,411 @@
+#!/usr/bin/env python
+"""
+Script to initialize an exercise for Exercism's Rust track
+
+Why a Python script in the Rust track repo? Distribution.
+A rust program would either need to be precompiled for various
+platforms, and available for download (in which case it wouldn't
+conveniently work in the repository), or would need to be included
+as a sub-crate and compiled locally. A python script can simply
+be a single file and depend on the user's system Python, if desired.
+
+This module requires Python3.5 or newer
+"""
+from __future__ import print_function

```

Comment:
My understanding is that this will have no effect on Python 3, but will allow this script to run and print its error messages on Python 2, when it instead would not be able to.

If this guess was correct, then there is no need to respond to it.